cmd/brew-services.rb

```diff
@@ -304,7 +304,11 @@ def start(target, custom_plist = nil)
     def stop(target)
       if target.is_a?(Service) && !target.loaded?
         rm target.dest if target.dest.exist? # get rid of installed plist anyway, dude
-        odie "Service `#{target.name}` not running, wanna start it? Try `#{bin} start #{target.name}`"
+        if target.started?
+          odie "Service `#{target.name}` is started as `#{target.started_as}`. Try `#{ServicesCli.root? ? "" : "sudo "}#{bin} stop #{target.name}`"
```

Comment:
`#{ServicesCli.root? ? "" : "sudo "}` could instead be `#{"sudo " if ServicesCli.root?}`